lib/plugins/Label_4A.test.ts

```diff
@@ -0,0 +1,158 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_4A } from './Label_4A';
+
+test('matches Label 4A qualifiers', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_4A(decoder);
+
+  expect(decoderPlugin.decode).toBeDefined();
+  expect(decoderPlugin.name).toBe('label-4a');
+  expect(decoderPlugin.qualifiers).toBeDefined();
+  expect(decoderPlugin.qualifiers()).toEqual({
+    labels: ['4A'],
+  });
+});
+
+test('decodes Label 4A, variant 1', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_4A(decoder);
```

Comment:
This is me being opinionated. Can we wrap all the tests in a `Label 4A` describe and have this be in a beforeEach (all?) so it doesn't need to be in every test?